src/SLCore.IntegrationTests/FileAnalysisTests.cs

```diff
@@ -108,6 +161,15 @@ private static string GetFullPath(string fileToAnalyzeRelativePath)
         return Path.GetFullPath(fileToAnalyzeRelativePath);
     }
 
+    private static void UpdateStandaloneRulesConfiguration(SLCoreTestRunner slCoreTestRunner, params KeyValuePair<string, StandaloneRuleConfigDto>[] ruleKeyToRuleConfigDto)
+    {
+        slCoreTestRunner.SLCoreServiceProvider.TryGetTransientService(out IRulesSLCoreService rulesCoreService)
+            .Should().BeTrue();
+        var ruleConfigByKey = ruleKeyToRuleConfigDto.ToDictionary(ruleKeyToDto => ruleKeyToDto.Key, keyValuePair => keyValuePair.Value);
+
+        rulesCoreService.UpdateStandaloneRulesConfiguration(new UpdateStandaloneRulesConfigurationParams(ruleConfigByKey));
```

Comment:
Minor: we can skip sending rule config if it's empty

Reply:
Done